mozilla_django_oidc/auth.py

```diff
@@ -124,16 +124,17 @@ def create_user(self, claims):
         username = self.get_username(claims)
 
         # Create user with custom values if they're specified
-        if not (self.OIDC_RP_UNIQUE_IDENTIFIER == self.OIDC_RP_UNIQUE_IDENTIFIER == 'email'):
+        if not ((self.OIDC_RP_UNIQUE_IDENTIFIER == self.OIDC_RP_UNIQUE_IDENTIFIER == 'email') or 
+            (self.OIDC_RP_UNIQUE_IDENTIFIER == self.OIDC_RP_UNIQUE_IDENTIFIER == 'username')):
```

Comment:
Is this `self.OIDC_RP_UNIQUE_IDENTIFIER == self.OIDC_RP_UNIQUE_IDENTIFIER` redundant?

Reply:
Maybe I meant to look at OP and RP?